conjure-java-jaxrs-client/build.gradle

```diff
@@ -4,7 +4,7 @@ apply plugin: 'com.palantir.revapi'
 dependencies {
     api project(":extras:refresh-utils")
     api project(":client-config")
-    api project(":okhttp-clients")
+    api project(":conjure-java-legacy-clients")
```

Comment:
Doesn't API mean consumers can still get it? I'm assuming this is a first PR in a chain

Reply:
Yep, this new module doesn't provide okhttp, rather two public classes that used to be provided by `okhttp-clients`: `HostEventsSink` (the deprecated version, long story) and `QosExceptionResponseMapper` which is shared between jaxrs and retrofit.